docs/getting-started/quickstart.md

```diff
@@ -0,0 +1,185 @@
+
+    <Tabs
+        defaultValue="macOS"
+        values={[
+        {label: 'macOS', value: 'macOS'},
+        {label: 'Linux', value: 'Linux'},
+        {label: 'Windows Subsystem for Linux (WSL)', value: 'Windows Subsystem for Linux (WSL)'},
+        {label: 'Docker', value: 'Docker'},
+        ]}
+    >
+
+    <TabItem value='macOS'>
+
+    1. Install the Semgrep CLI and confirm the installation:
+
+        ```console
+        # with homebrew
+        brew install semgrep
+
+        # with pip
+        python3 -m pip install semgrep
+
+        # confirm
+        semgrep --version
+        ```
+
+    2. Log in to your Semgrep account (running this command will launch a browser window, but you can also use the link that's returned in the CLI to proceed):
+
+        ```console
+        semgrep login
+        ```
+
+    3. In the **Semgrep CLI login**, click **Activate** to proceed.
+
+    4. Return to the CLI, navigate to the root of your repo, and run your first scan:
+
+        ```console
+        semgrep ci
+        ```
+
+    </TabItem>
+
+    <TabItem value='Linux'>
+
+    1. Install the Semgrep CLI and confirm the installation:
+
+        ```console
+        # install
+        python3 -m pip install semgrep
+
+        # confirm
+        semgrep --version
+        ```
+
+    2. Log in to your Semgrep account (running this command will launch a browser window, but you can also use the link that's returned in the CLI to proceed):
+
+        ```console
+        semgrep login
+        ```
+
+    3. In the **Semgrep CLI login**, click **Activate** to proceed.
+
+    4. Return to the CLI, navigate to the root of your repo, and run your first scan:
+
+        ```console
+        semgrep ci
+        ```
+
+    </TabItem>
+
+    <TabItem value='Windows Subsystem for Linux (WSL)'>
+
+    :::info Prerequisites
+    Ensure that you have the [Windows Subsystem for Linux installed](https://learn.microsoft.com/en-us/windows/wsl/install) before proceeding.
+    :::
+
+    1. Using the WSL interface, install the Semgrep CLI and confirm the installation:
+
+        ```console
+        #install
+        python3 -m pip install semgrep
+
+        # confirm
+        semgrep --version
+        ```
+
+    2. Log in to your Semgrep account (running this command will launch a browser window, but you can also use the link that's returned in the CLI to proceed):
+
+        ```console
+        semgrep login
+        ```
+    
+    3. In the **Semgrep CLI login**, click **Activate** to proceed.
+    
+    4. Return to the WSL interface, navigate to the root of your repo, and run your first scan:
+
+        ```console
+        semgrep ci
+        ```
+
+    </TabItem>
+
+    <TabItem value='Docker'>
+
+    :::info Prerequisites
+    Ensure that you have the [Docker installed](https://docs.docker.com/desktop/) before proceeding.
```

Comment:
```suggestion
    Ensure that you have [Docker installed](https://docs.docker.com/desktop/) before proceeding.
```
Yet another proper noun, so we don't use the. Neither does Docker for their docs. [https://docs.docker.com/engine/install/](https://docs.docker.com/engine/install/)